example/revisions.trig

```diff
@@ -0,0 +1,78 @@
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix exdata: <https://example.com/data/>.
+@prefix rec: <https://rdf.equinor.com/ontology/record/>.
+@prefix rev: <https://rdf.equinor.com/ontology/revision/>.
+@prefix famdata: <https://rdf.equinor.com/fam/>.
+@prefix contract: <https://rdf.equinor.com/fam/Contract/>.
+@prefix document: <https://rdf.equinor.com/fam/Document/>.
+@prefix facility: <https://rdf.equinor.com/fam/Facility/>.
+@prefix project: <https://rdf.equinor.com/fam/Project/>.
+@prefix fsystem: <https://rdf.equinor.com/fam/FunctionalSystem/>.
+@prefix famvocab: <https://rdf.equinor.com/ontology/fam/v1/>.
+@prefix prov: <http://www.w3.org/ns/prov#>.
+
+exdata:RecordID123 {
+    exdata:RecordID123 a rec:Record;
+                        rdfs:comment "Container record, defines scope for subrecords"^^xsd:string;
+                        prov:generatedAtTime "2022-11-03"^^xsd:date;
+                        rec:isInScope contract:12345678,
+                                    document:B123-EX-W-LA-00001,
+                                    facility:EXMPL,
+                                    project:L.01234A.001.
+}
+
+exdata:RecordID123.0 {
+    exdata:RecordID123.0 a rec:Record;
+                        rdfs:comment "First MEL delivered on Example Facility"^^xsd:string;
+                        prov:generatedAtTime "2022-11-03"^^xsd:date;
+                        rec:describes document:B123-EX-W-LA-00001;
+                        rec:isSubRecordOf exdata:RecordID123.
+    document:B123-EX-W-LA-00001 a <https://rdf.equinor.com/ontology/mel/EquipmentList>;
+                                rdfs:label "B123-EX-W-LA-00001"^^xsd:string;
+                                famvocab:DocumentType "LA"^^xsd:string;
+                                famvocab:Facility facility:EXMPL;
+                                famvocab:FunctionalSystem fsystem:ENG;
+                                famvocab:Location "Northsea"^^xsd:string;
+                                famvocab:Title "MASTER EQUIPMENT LIST - MEL"^^xsd:string.
+}
+
+exdata:RecordID123.1 {
+    exdata:RecordID123.1 a rec:Record;
+                        prov:generatedAtTime "2022-11-03"^^xsd:date;
+                        rec:describes document:B123-EX-W-LA-00001.F01;
+                        rec:isSubRecordOf exdata:RecordID123.
+    document:B123-EX-W-LA-00001.F01 a rev:DocumentRevision;
+                                    rdfs:label "First delivered revision of MEL"^^xsd:string;
+                                    rev:revisionNumber "F01"^^xsd:string;
+                                    prov:generatedAtTime "2022-11-03"^^xsd:date;
+                                    rev:issuedBy "Kari Nordkvinne"^^xsd:string;
+                                    rev:isNewRevisionOf document:B123-EX-W-LA-00001.A01;
+                                    famvocab:DisciplineApprovedBy "NM"^^xsd:string;
```

Comment:
rev:IssuedBy and famvocab:DisciplineApprovedBy is not information that is currently part of the Tie Message. Include for later use?

Reply:
These are just meant as examples of "extra data" on a revision